README.md

```diff
@@ -101,6 +101,62 @@ add `priority` label when you start backing us :smile:
 
 See: [docs/deployment.md](docs/deployment.md)
 
+# Development
+
+> This abbreviated guide is meant to give a guide as to how github apps are made and basic steps to reproduce the OpenCollective Bot should you wish to make futher changes.
+
+Visit the [Github App Docs](https://developer.github.com/apps/quickstart-guides/setting-up-your-development-environment/) for a full documentation on Github Apps
+
+## Installation
+
+Feel free to clone or download this project using the **git clone** and install using **Yarn install** to install required dependencies
+
+**WebHooks** are crucial to production **Github Apps**. Hence, you can use [Ngrok](https://ngrok.com) , [Smee](https://smee.io) or other comfortable options .
+
+> **Smee** which is supported by **Probot** is however used in this guide .
+
+### Setting up Smee
+
+> Visit the [Smee](https://smee.io) website, to start a new channel
+
+-After creating a channel, take note of the **Webhook Proxy URL** shown as it would be used as the **Webhook Url** in the about page when resgistering the app on github
+
+- `npm i -g smee-client` to install the **smee client**
+
+### Registering App
+
+- Follow steps listed at [Github](https://developer.github.com/apps/building-github-apps/creating-a-github-app/) to register your bot.
+
+- For the **Homepage URL**, use the **Webhook Proxy URL** generated by Smee
+
+- After a succesfull registration, generate a `private key`(file with a `.pem` extension ), rename to `private.pem` and store in the root project folder .
+
+After registering the app, take note of the **App ID** shown in the about page
+
+> Settings used can be changed later, incase of any changes such as the **Webhook URL**
+
+#### Credentials
+
+> `Dotenv-cli` is used to handle credentials within this project
+
+- Create a `.env` file in the working directory with the following variabes below
+
+```js
+ APP_ID = <App ID>
+ WEBHOOK_SECRET = development
+ PRIVATE_KEY_PATH = < `.pem` file placed in the `cwd`> e.g `PRIVATE_KEY_PATH = "private.pem"`
+```
+
+- **APP_ID** takes the App_id found at the **about** page.
+
+### Installation and Inspecting
```

Comment:
We already have an "Installation" step, at this point this is not about installation.